### PR TITLE
Revert "Revert "Add proof-of-existance tutorial""

### PR DESCRIPTION
**Pull request by JakeSCahill**
Reverts iotaledger/documentation#440

### CODE REVIEW

File: utils/0.1/official/proof-of-existance/overview.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
+# Proof of existance overview
+
+**Files are an important means of transporting information such as contracts among different parties. Sometimes, one party may change a file without the others knowing. So, all parties need a way to reliably prove that a file has not been changed from an established state to be able to trust the file. To prove that a file is unchanged, you can use the proof-of-existance utility to hash the file's contents and attach it to the Tangle. This way, any party can later hash the file and compare it to the one on the Tangle. If the hashes are the same, the file is unchanged.**
+
```
Comment (mtrabelsi) [inline utils/0.1/official/proof-of-existance/overview.md]: I thought i will be nice to mention that the Tangle Data is Immutable, and we use it as a source of truth.  What do you think?

File: utils/0.1/official/proof-of-existance/overview.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    ```
+
+3. Create an `index.js` file in the same directory as the `contract.txt` file, and start by requiring the libraries
+
+    ```js
+    require('regenerator-runtime');
+    const PoEx = require ('@iota/poex-tool');
+    ```
+
+4. Create an asynchronous function that takes a path to a file whose hash you want to attach to the Tangle
```
Comment (mtrabelsi) [inline utils/0.1/official/proof-of-existance/overview.md]: we should mention also the other option that PoEx also works with Binary data too ( not only path )

File: utils/0.1/official/proof-of-existance/overview.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+    const hash = PoEx.hash(file);
+
+        try {
+            // Attach the file hash to the Tangle in a bundle
+            const bundle = await PoEx.publish({
+            provider: 'https://nodes.devnet.iota.org:443',
+            data: hash,
+            tag: 'POEXTUTORIAL',
+            address: 'HELLOWORLDHELLOWORLDHELLOWORLDHELLOWORLDHELLOWORLDHELLOWORLDHELLOWORLDHELLOWORLDD',
```
Comment (mtrabelsi) [inline utils/0.1/official/proof-of-existance/overview.md]: if these address/seed are omitted it will fall back to some defaults one

Comment (JakeSCahill) [reply utils/0.1/official/proof-of-existance/overview.md]: I'm not sure we want to advertise that. We give some example values here, just so users can see which ones they need to provide, but also test the code with some placeholders.

File: utils/0.1/official/proof-of-existance/overview.md (hunk 2 above)
Comment (JakeSCahill) [reply utils/0.1/official/proof-of-existance/overview.md]: Added a note about this

**Pull request merged**